_posts/2016-09-30-subtype-typeclasses.md

```diff
@@ -89,13 +95,13 @@ def bar[F[_]: Applicative: Functor]: F[Int] = Applicative[F].pure(10)
 ```
 
 ```scala
-def callBar[F[_]: Monad: Traverse]: F[Int] = bar[F]
-// <console>:19: error: ambiguous implicit values:
-//  both value evidence$2 of type cats.Traverse[F]
-//  and value evidence$1 of type cats.Monad[F]
-//  match expected type cats.Functor[F]
-//        def callBar[F[_]: Monad: Traverse]: F[Int] = bar[F]
-//                                                        ^
+scala> def callBar[F[_]: Monad: Traverse]: F[Int] = bar[F]
```

Comment:
One possibility is to:
1. move all posts to the `_tut` directory
2. add `_posts` to `.gitignore`
3. copy non-tut files from `_tut` to `_posts`
4. process tut files from `_tut` to `_posts`
5. let Travis commit the results in the `gh-pages` branch